### PR TITLE
XMAS-5631 - Add npm package creation

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Description:
- See ticket description

#### Testing:
- Verify that the artifacts contain the following two files:
     - build/ixbrl-viewer-0.0.0.tgz and that it has type npm
     - build/dist/ixbrl_viewer-0.0.0.tar.gz and that it has type pypi

**review**:
@Workiva/xmas

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,14 @@ RUN pip install -r requirements-dev.txt
 WORKDIR /build/
 ADD . /build/
 
+# The following command replaces the @VERSION@ string in setup.py and package.json
+# with the tagged version number from GIT_TAG
+ENV VERSION="0.0.0"
+RUN if [ "$GIT_TAG" != "" ] ; then VERSION=$GIT_TAG ; fi
+RUN echo Version = $VERSION
+RUN sed -i s/@VERSION@/$VERSION/ setup.py
+RUN sed -i s/@VERSION@/$VERSION/ package.json
```
Comment (brettkail-wk) [inline Dockerfile]: Perhaps:
```
RUN sed -i s/@VERSION@/$VERSION/ setup.py package.json
```

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,14 @@ RUN pip install -r requirements-dev.txt
 WORKDIR /build/
 ADD . /build/
 
+# The following command replaces the @VERSION@ string in setup.py and package.json
+# with the tagged version number from GIT_TAG
+ENV VERSION="0.0.0"
```
Comment (brettkail-wk) [inline Dockerfile]: Is 0.0.0 better than the existing 0.0.1?

Comment (derekgengenbacher-wf) [reply Dockerfile]: I wanted to be explicitly clear that this was not an actual version.

**Pull request merged**